build(bake): specify local and remote bake definitions (#18711)
docker/docs: content/build/bake/remote-definition.md

#8 0.136 -rwxrwxrwx    1 root     root          9620 Jul 27 18:31 vendor.conf
#8 0.136 /bin/sh: stop: not found
```

> 

Reviewer: I think we could start a new section here

```suggestion

## Remote definition with the --file flag

```

> You can also specify the Bake definition to load from the remote repository:

Reviewer: ```suggestion
You can also specify the Bake definition to load from the remote repository,
using the `--file` or `-f` flag:
```

```console
docker buildx bake -f bake.hcl "https://github.com/crazy-max/buildx.git#remote-with-local"
```

```text
...
#4 [2/2] RUN echo "hello world"
#4 0.270 hello world
#4 DONE 0.3s
```

> And load a local definition using the `cwd://` prefix with a remote one as well:

Reviewer: ```suggestion
If you want to use a combination of local and remote definitions,
you can specify a local definition using the `cwd://` prefix with `-f`:
```
